cpt/test/integration/basic_test.py

```diff
@@ -310,6 +310,26 @@ def configure(self):
                 json_data = json.load(json_content)
                 self.assertFalse(json_data[0]["package"]["error"])
 
+    def test_disable_test_folder(self):
+        conanfile = """from conans import ConanFile
+
+class Pkg(ConanFile):
+    name = "lib"
+    version = "1.0"
+"""
+        self.save_conanfile(conanfile)
+        conanfile = """from conans import ConanFile
+
+class Pkg(ConanFile):
+    def test(self):
+        raise Exception("Should not run")
+"""
+        tools.save(os.path.join(self.tmp_folder, "test_package", "conanfile.py"), conanfile)
+        with tools.environment_append({"CPT_TEST_FOLDER": "False"}):
+            self.packager = ConanMultiPackager(out=self.output.write)
+            self.packager.add_common_builds()
+            self.packager.run()
```

Comment:
can't you check the output to verify it doesn't have `Should not run`?

Reply:
actually is not necessary, because it raises an exception and there is not treatment, any error/exception results in complete fail. If you check, most of tests in BasicTest follow the same approach.

Reply:
@danimtb I just added a new test which simulates an invalid test folder. It covers your comment and also test regressions to CPT_TEST_FOLDER